### PR TITLE
[DOCU-385] Dev Portal styling doesn't carry over

### PR DESCRIPTION
**Pull request by falondarville**
### Summary

Add a note that on-prem Dev Portal doesn't carry over between default Dev Portal and other Workspaces.

### Reason

Addresses [DOCU-385](https://konghq.atlassian.net/browse/DOCU-385)

### Testing

https://deploy-preview-3675--kongdocs.netlify.app/gateway/2.7.x/developer-portal/theme-customization/easy-theme-editing/#the-appearance-tab

### CODE REVIEW

File: app/gateway/2.7.x/developer-portal/theme-customization/easy-theme-editing.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ The Kong Developer Portal ships with a default theme, including preset images, b
 
 ## The Appearance Tab
 
+{:.note}
+> **Note**: Styling does not automatically carry over from the default Dev Portal to different Dev Portals in other Workspaces. Use the [Dev Portal CLI](/gateway/latest/developer-portal/helpers/cli) to manually copy templates to Workspaces where you want to duplicate the styling.
```
Comment (lena-larionova) [inline app/gateway/2.7.x/developer-portal/theme-customization/easy-theme-editing.md:16]:
```suggestion
> **Note**: Styling does not automatically carry over from the default Dev Portal to different Dev Portals in other workspaces. Use the [Dev Portal CLI](/gateway/latest/developer-portal/helpers/cli) to manually copy templates to workspaces where you want to duplicate the styling.
```

**Review by lena-larionova — COMMENTED** [review]
This note is useful, but I think a short task/list of direction on how to actually use the CLI to copy the templates would be useful as well. 

I took a quick look at the docs and no obvious place for that content stood out to me, so maybe a short topic under "Customization" would be the way to go? Name it something like "Migrating templates between workspaces" and use [Karen's instructions](https://konghq.atlassian.net/browse/FTI-2207?focusedCommentId=36358) to apply the templates.

**Comment by falondarville** [comment]
@lena-larionova let's create a new task for your new page suggestion.

**Review by lena-larionova — APPROVED** [review]

**Comment by falondarville** [comment]
Task created: https://konghq.atlassian.net/browse/DOCU-2198

**Pull request merged**